source/index.html.md

```diff
@@ -292,7 +292,8 @@ Choose a number or apply filter (format: [groupId:]artifactId, case sensitive co
 Choose com.snaplogic.tools:SnapArchetype version: 
 1: 35.0
 2: 36.0
-Choose a number: 2:
+3: 37.0
```

Comment:
Can remove 35.0 from the archetype catalog.

Reply:
I guess this is a good question, do we want to remove versions to have only 2 versions available at a time? we previously didn't do that, but I cleared a large number of them out recently.

Reply:
Yes, we should only keep the active versions going forward.

Reply:
alright, removed 35.0 from the list in the log, I'll handle this when the Archetype items are updated, approved, and deployed.